lib/plugins/aws/provider.js

```diff
@@ -1230,6 +1230,13 @@ class AwsProvider {
             tracing: { $ref: '#/definitions/awsLambdaTracing' },
             versionFunction: { $ref: '#/definitions/awsLambdaVersioning' },
             vpc: { $ref: '#/definitions/awsLambdaVpcConfig' },
+            httpApi: {
+              type: 'object',
+              properties: {
+                payload: { type: 'string', enum: ['1.0', '2.0'] },
```

Comment:
let's stick to `payloadVersion` variable name to not introduce discrepancies between `provider.httpApi` and `function.httpApi` as it might be confusing to users that properties controlling the same thing are called differently

Reply:
Additionally, let's move the `payloadVersion` schema to a definition so it can be reused - it will help us ensure to not omit upading it in all places when we will need to change that definition in the future